docker/pdo_client.dockerfile

```diff
@@ -65,13 +65,13 @@ COPY --chown=${UNAME}:${UNAME} tools/*.sh ./
 # build it!!!
 RUN /project/pdo/tools/build_client.sh
 
-ARG PDO_HOSTNAME
-ENV PDO_HOSTNAME=$PDO_HOSTNAME
+ARG PDO_HOSTNAME=localhost
+ENV PDO_HOSTNAME=${PDO_HOSTNAME}
 
-ARG PDO_LEDGER_URL
-ENV PDO_LEDGER_URL=$PDO_LEDGER_URL
```

Comment:
Thinking about this, it seems appropriate not to include a ledger url in the image, but I am wondering if even the hostname should be provided at a later time.

Reply:
I see your point. I think a valid PDO_HOSTNAME var s required by some of our build scripts. That said, in that case we could (should) drop the ARG part but still retain the ENV so the build works?

Reply:
I'm writing up a more complete version of what needs to happen with docker. will push in an issue for discussion. We (mea culpa) are still mixing up 1) a generic image that anyone could deploy on any network and 2) an image pre-configured for a specific network. I think we should make ALL images that we build preconfigured with hostname as localhost and ledger as 127.0.0.1:6600. Then use the start_* scripts to set what actually needs to be done. Keys are already provided along similar lines.

Reply:
> I thoutht i could split check_pdo_env in runtime and build-time check but alas with some tests scripts during build it willl also be required. 

which scripts do you think we run at build time? specifically for docker...